.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+# PR Description
```

Comment:
Can you change the heading of the title here. Instead of using h1 (#) please change to h3 which allow reading other content

This is minor changes

Reply:
Solved.